motimo-api/src/main/java/kr/co/api/goal/rqrs/GoalCreateRq.java

```diff
@@ -0,0 +1,38 @@
+package kr.co.api.goal.rqrs;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.annotation.Nullable;
+import jakarta.validation.constraints.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public record GoalCreateRq(
+    @NotNull
+    @Size(min = 1, max = 20)
+    @Schema(description = "목표 이름", example = "자격증 따기")
+    String title,
+
+    @NotNull
+    @Schema(description = "개월 수로 기간 설정 여부")
+    boolean isMonthSetting,
+
+    @Size(min = 1, max = 12)
+    @Schema(description = "목표 개월 수")
+    Integer month,
+
+    @Schema(description = "목표 완료 날짜", format = "date")
+    LocalDate dueDate,
+
+    @Schema(description = "세부 목표 목록")
+    List<SubGoalCreateRq> subGoals
+) {
+    public GoalCreateRq {
+        if (Objects.isNull(subGoals)) {
+            subGoals = new ArrayList<>();
+        }
+    }
+}
```

Comment:
```suggestion
    @Schema(description = "세부 목표 목록")
    @JsonSetter(nulls = Nulls.AS_EMPTY)
    List<SubGoalCreateRq> subGoals
) {
}
```

요거 subGoals가 `null`로 들어올 경우를 방지하기 위해 생성자에서 null 체크 후 빈 리스트로 초기화하신 부분인 것 같아요!@!
이런 경우에는 `@JsonSetter(nulls = Nulls.AS_EMPTY)` 어노테이션 사용할 수도 있을 거 같아요@!

Reply:
감사합니다!.!